src/apiClients/AirTableApiClient.ts

```diff
@@ -78,6 +89,38 @@ class AirTableApiClient {
 
     return organizations;
   }
+
+  async logError(error: ErrorObject): Promise<boolean> {
+    const promise: Promise<boolean> = new Promise(async resolve => {
+      try {
+        await this.base(BASE_NAMES.ERRORS).create([error]);
+
+        const allRecords = await this.base(BASE_NAMES.ERRORS)
+          .select({ view: 'Grid view' })
+          .all();
+
+        if (allRecords.length > MAXERRORLENGTH) {
+          let deleteSet: string[] = [];
+          for (let i = 0; i < 10; i++) {
+            deleteSet.push(allRecords[i].id);
+          }
+          await this.base(BASE_NAMES.ERRORS).destroy(deleteSet);
+
+          deleteSet = [];
+          for (let i = 10; i < 20; i++) {
+            deleteSet.push(allRecords[i].id);
+          }
```

Comment:
See if this works: you can do `deleteSet = allRecords.slice(10,20).map(record => record.id)`